apps/website/src/app/components/Footer.tsx

```diff
@@ -10,21 +11,54 @@ export const Footer = () => {
         &copy; NameHash Labs. All Rights Reserved
       </p>
       <div className="flex flex-row justify-between gap-3">
-        <a href={"https://twitter.com/NamehashLabs"}>
-          <TwitterIcon />
+        <a
+          href="https://twitter.com/NamehashLabs"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <TwitterIcon
+            className={
+              "gt_mobile:transition gt_mobile:hover:fill-black text-[#AFAFAF] fill-current transition-all duration-200"
+            }
+          />
         </a>
-        <a href={"https://github.com/namehash/nameguard"}>
-          <GithubHoverableLogo />
+        <a
+          href="https://github.com/namehash/nameguard"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <GithubLogo
+            className={
```

Comment:
This can just be `className="..."`

Applies to other files

Reply:
Understood. I'll take a look in the whole directory and correct it where possible

Reply:
I changed a few of these instances on the homepage, so no changes needed there. Thanks 🙏🏻